src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const pricePerDay = 40;
+  const fullPrise = days * pricePerDay;
+  const cashbackForTreeDays = 20;
+  const cashbackForSevenDays = 50;
```

Comment:
[Checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md) 5th
item, don't hardcode any values in variable names, value might change, use smth like `basicDiscount`, `advancedDiscount`

Reply:
Fixed